demisto_sdk/__main__.py

```diff
@@ -1474,6 +1474,13 @@ def upload(ctx, **kwargs):
     is_flag=True,
     default=False,
 )
+@click.option(
+    "-aruuids",
+    "--auto-replace-uuids",
+    help="Whether to run demisto-sdk format on downloaded files or not",
+    is_flag=True,
+)
+
```

Comment:
I think that we should add the opposite flag - 'no-uuid-replacement' (or something like that..) which setting to True will avoid UUID replacement since we don't want to affect the existing functionality (which is defaultly change the UUID).

Let's change that and change the other files accordingly.

Reply:
we can have both, but default=on. See other flags, we usually do `--do-thing/--no-do-thing`

Reply:
I understand but according to dan in the ticket:
'we should have a boolean flag auto-replace-uuids which defaults to true for BC, and for the customer's needs, they will set it to false.'

Reply:
changing it to replace_uuids